### PR TITLE
US#156748 external survey mashup

### PR DESCRIPTION
**Pull request by gpolunin**

### CODE REVIEW

File: External Survey/ExternalSurvey.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+tau.mashups.addModule("external-survey-mashup/config", {
+	link: 'https://docs.google.com/a/targetprocess.com/forms/d/e/1FAIpQLSf0MKuIwlgiAeHAxkhbkroD6h2z4Ige-3AfQJrSFM-XcblcMA/viewform?${entry.187418020}&${entry.2079944653}&entry.518446863&entry.1358339087&entry.1690215969&entry.1500006626&entry.613608536',
+	linkLabel: 'Answer few questions',
```
Comment (pogotskaya) [inline External Survey/ExternalSurvey.config.js]: Answer _a_ few questions

File: External Survey/ExternalSurvey.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	text: 'Help us out by answering a few questions. We’ll appreciate it a lot.',
+	header: 'Want better navigation in Targetprocess?',
+	clientStorageKey: "navigation",
```
Comment (victor-homyakov) [inline External Survey/ExternalSurvey.config.js]: Code style: use single quotes

File: External Survey/ExternalSurvey.config.js (hunk 1 above)
Comment (victor-homyakov) [inline External Survey/ExternalSurvey.config.js]: And don't forget to re-generate screenshot

File: External Survey/ExternalSurvey.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                    header: this._config.header
+
+                });
+
+                var onShow = function () {
+                    $('#' + linkId).click(this._saveShownFlag.bind(this));
+                }.bind(this);
+
+                var onHide = function () {
+                    $('#' + linkId).unbind('click');
```
Comment (victor-homyakov) [inline External Survey/ExternalSurvey.js]: Deprecated API. Use `.off()` instead of `.unbind()`.

File: External Survey/ExternalSurvey.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+                    link: this._getLink(),
+                    linkLabel: this._config.linkLabel,
+                    linkId: linkId,
+                    text: this._config.text,
+                    header: this._config.header
+
+                });
+
+                var onShow = function () {
+                    $('#' + linkId).click(this._saveShownFlag.bind(this));
```
Comment (victor-homyakov) [inline External Survey/ExternalSurvey.js]: Use `.on('click', ...)`

**Comment by gpolunin** [comment]
Pull request moved to private mashup repo here:
https://github.com/TargetProcess/MashupsPrivateLibrary/pull/6

File: External Survey/ExternalSurvey.js (hunk 3 above)
Comment (gpolunin) [inline External Survey/ExternalSurvey.js]: Done

File: External Survey/ExternalSurvey.js (hunk 4 above)
Comment (gpolunin) [inline External Survey/ExternalSurvey.js]: Done

File: External Survey/ExternalSurvey.config.js (hunk 2 above)
Comment (gpolunin) [inline External Survey/ExternalSurvey.config.js]: Done

File: External Survey/ExternalSurvey.config.js (hunk 1 above)
Comment (gpolunin) [inline External Survey/ExternalSurvey.config.js]: Done

**Pull request closed without merging**